book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  # Other methods of Book class
```

Comment:
- [ ] For this class, a method for adding rentals is required so rentals can be added. Kindly add that 👍🏽